lighthouse-core/test/audits/byte-efficiency/uses-long-cache-ttl-test.js

```diff
@@ -103,11 +103,10 @@ describe('Cache headers audit', () => {
 
     const networkRecords = [
       networkRecord({headers: {
-        'cache-control': 'must-revalidate,max-age=3600',
```

Comment:
can we come up with a different directive to stick in here to make sure we parse properly :)

Reply:
ya